prelexer.cpp

```diff
@@ -423,6 +423,9 @@ namespace Sass {
     const char* number(const char* src) {
       return sequence< optional<sign>, unsigned_number>(src);
     }
+    const char* sass_number(const char* src) {
+      return sequence< optional<sign>, unsigned_number, optional < identifier > >(src);
```

Comment:
Interesting. It'd be worth looking at the Ruby parse to see what their rule is but for now :+1: